### PR TITLE
Remove PII features

### PR DESCRIPTION
**Pull request by csh519**
## Description

The team leader decided to remove PII related features from CredSweeper, so PII related features will be removed.

- Remove PII rule
- Remove Phone rule
- Remove IBAN rule
- Remove VIN rule
- Remove Cred Card Number rule

## How has this been tested?

- [X] UnitTest

### CODE REVIEW

**Review by babenek — APPROVED** [review]
Okay

**Review by xDizzix — APPROVED** [review]
+

**Review by Yullia — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 CredSweeper is a tool to detect credentials in any directories or files.
 CredSweeper could help users to detect unwanted exposure of credentials
-(such as personal information, token, passwords, api keys etc.) in advance.
+(such as token, passwords, api keys etc.) in advance.
```
Comment (kmnls) [inline README.md:37]:
```suggestion
(such as tokens, passwords, api keys etc.) in advance.
```

**Review by kmnls — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (babenek) [reply README.md:37]: Will be added in next pr

Comment (csh519) [reply README.md:37]: Okay thank you.

**Pull request merged**